doc/userguide/output/lua-output.rst

```diff
@@ -34,7 +34,7 @@ Example:
   end
 
   function setup (args)
-      filename = SCLogPath() .. "/" .. name
+      filename = SCLogPath() .. "example.log"
```

Comment:
I don't think is is correct.  We have S-V tests that use the first form just fine.  If you leave a trailing '/' in your log directory configuration, then yes, `filename` will end up with a double `//` in it, but is this a problem?  If your log directory configuration does not have a trailing `/` in it (which is typical of a default install) then this new form will fail.

Reply:
ok
but what about _empty_ 'name' variable in this script?

Reply:
> but what about _empty_ 'name' variable in this script?

Yeah, the probably should be fixed.